src/igvfd/audit/measurement_set.py

```diff
@@ -88,9 +96,18 @@ def audit_unspecified_protocol(value, system):
 @audit_checker('MeasurementSet', frame='object')
 def audit_inconsistent_readout(value, system):
     '''
-        audit_detail: CRISPR-based and MPRA assays are required to specify a readout that is different from the assay term of the measurement set, other assays should not include readout specification.
-        audit_category: inconsistent readout
-        audit_levels: ERROR, NOT_COMPLIANT
+    [
+        {
+            "audit_description": "CRISPR-based and MPRA assays are required to specify a readout, other assays should not include readout specification.",
```

Comment:
Would not the two entries be potentially separated on the Audits documentation UI? The second description is not mentioning assays and it maybe unclear?

Reply:
I don't really understand how it is unclear, if a readout is specified it should be different than the assay term regardless of if the assay expects a readout or not.

Reply:
what is your suggestion otherwise? To duplicate the description?